x/axelarnet/keeper/keeper.go

```diff
@@ -20,18 +20,26 @@ import (
 )
 
 var (
-	cosmosChainPrefix = key.FromStr("cosmos_chain")
-	feeCollector      = key.FromStr("fee_collector")
+	cosmosChainPrefix = key.FromStr("cosmos_chain")  // Deprecated: migrate to cosmosChainPrefixNew
+	feeCollector      = key.FromStr("fee_collector") // Deprecated: migrate to feeCollectorNew
 
-	transferPrefix       = key.FromStr("ibc_transfer")
-	ibcTransferQueueName = "route_transfer_queue"
-	// nonceKey is deprecated in v0.23
-	// nonceKey = key.FromUInt[uint64](1) <-- keeping this here so the number doesn't get reused
+	transferPrefix       = key.FromStr("ibc_transfer") // Deprecated: migrate to transferPrefixNew
+	ibcTransferQueueName = "route_transfer_queue"      // Deprecated: migrate to ibcTransferQueueNameNew
 
 	// failedTransferPrefix is deprecated in v0.23
-	failedTransferPrefix = key.FromUInt[uint64](2)
-	seqIDMappingPrefix   = key.FromUInt[uint64](3)
-	ibcPathPrefix        = key.FromUInt[uint64](4)
+	//TODO: what do we do with this?
```

Comment:
```suggestion
	// TODO: what do we do with this?
```

Reply:
Taking a page from protobuf, what about something like `key.Reserve(types.ModuleName, 2)` that does not return anything but reserves an entry in the map to prevent re-use?

Reply:
see below `_ = key.RegisterStaticKey(types.ModuleName, 1)` has the same effect.
The todo is here, because the key is actually still used but apparently deprecated, so what's the deal?

Reply:
ah sry I misunderstood. It appears to me that all functions using this prefix are not being anywhere though. Should be safe to "remove".